companyweb_base/models/res_partner.py

```diff
@@ -547,15 +551,16 @@ def cweb_button_copy_address(self):
             message=_("Companyweb Address Successfully copied")
         )
 
-    def _cweb_call_wizard_credentials(self, wizard_name):
+    @api.model
+    def _cweb_call_wizard_credentials(self, wizard_name, context):
         wizard_form = self.env.ref("companyweb_base.companyweb_credential_wizard")
         return {
             "name": wizard_name,
             "type": "ir.actions.act_window",
             "view_type": "form",
             "view_mode": "form",
-            "res_model": "companyweb_base.credential_wizard",
+            "res_model": "companyweb_base.credential_wizard_base",
             "view_id": wizard_form.id,
             "target": "new",
-            "context": self.env.context,
+            "context": context,
```

Comment:
Accepting the context as method argument sounds weird. Why not use `self.env.context` ?